doc/ref_cert/RC2/index.rst

```diff
@@ -0,0 +1,57 @@
+`<< Back <../>`__
+
+Reference Conformance - Kubernetes based platform for Telecom
+=============================================================
+
+This is the Kubernetes Based Reference Conformance (RC-2)
+
+Release Information
+-------------------
+
+**Bundle: 6**
+
+**Version: 0**
+
+**Release Date: 1st July 2021**
+
+Bundle/Version History
+----------------------
+
+============== ============= ===================
+Bundle.Version Date          Note
+============== ============= ===================
+0.0            15th May 2020 First Initial Draft
+4.0            25th Sep 2020 Baraque Release
+5.0            29th Jan 2021 Elbrus Release
+6.0            1st Jul 2021  Kali Release
+============== ============= ===================
+
+Overall Status
+--------------
+
++-----------------------------------+-----------------------------------+
+| Chapter                           | Status                            |
++===================================+===================================+
+| Chapter 01 - Introduction         | Lots of SME feedback              |
++-----------------------------------+-----------------------------------+
+| Chapter 02 - Kubernetes Test      | Lots of SME feedback              |
+| Cases and Requirements            |                                   |
+| Traceability                      |                                   |
++-----------------------------------+-----------------------------------+
+| Chapter 03 - Kubernetes Testing   | Dickering over fine points        |
+| Cookbook                          |                                   |
++-----------------------------------+-----------------------------------+
+| Chapter 04 - CNF Test Cases and   | Initial Framework Only            |
+| Requirements Traceability         |                                   |
++-----------------------------------+-----------------------------------+
+| Chapter 05 - CNF Testing Cookbook | Initial Framework Only            |
++-----------------------------------+-----------------------------------+
+| Chapter 06 - Gap analysis and     | Initial Framework Only            |
+| Development                       |                                   |
++-----------------------------------+-----------------------------------+
+
+.. toctree::
```

Comment:
Didn't generate the TOC

Reply:
yes it does when called by sphinx (you can simply call tox to install deps in a venv)